utils/src/main/scala/ai/h2o/sparkling/utils/SparkSessionUtils.scala

```diff
@@ -0,0 +1,30 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package ai.h2o.sparkling.utils
+
+import org.apache.spark.sql.SparkSession
+
+/**
+ * Internal utilities methods for Spark Session
+ */
+object SparkSessionUtils {
+  def active: SparkSession = {
+    SparkSession.clearActiveSession
```

Comment:
if `SparkSession.clearActiveSession` is called before `SparkSession.builder().getOrCreate()`, doesn't it cause that getOrCreate() creates a new Spark session each time?

Reply:
`clearActiveSession` this method just clears currently set session in the active thread, so the getOrCreate can return the global session.

But will be removed anyway, I was trying to debug why pysparkling is failing on Spark 2.1 on this change & this wasn't it